docs/pages/SPA-vs-SSR.page.server.mdx

```diff
@@ -6,7 +6,7 @@ Modern UI frameworks (React/Vue/...) enable a wide range of render modes:
  - Pre-rendering (aka SSG)
  - HTML-only
 
-We compare these render modes with each other and explain which render mode should be used when.
+We compare these render modes with each other and explain when which render mode should be used.
```

Comment:
How about this?
> We compare render modes and explain when to use each mode.

Reply:
I agree it sounds much more natural 👍 I'll update the line.